src/content/blog/IntroOSU.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Tips and Tricks For Success"
+description: "A quick start to how to succeed here at OSU in EECS!"
+coverImage: "/Oregon-State-Beavers-logo.png"
+category: "Learning"
+inDev: False
+pubDate: "2025-09-28"
+updatedDate: "2025-09-28"
+publishDate: "Sept 28th"
+---
+# Welcome to CS at Oregon State!
+
+## Intro
+
+Hello everyone! For those new and returning students welcome to/welcome back to Oregon State! To get right to it, the purpose of this post is to simply be a rough guide on where to go for learning resources, how to set yourself of for success for internships, as well as providing links for the most common tools you'll need during your time here. With that, let's jump right to it.
+
+## Internships 
+On top of everyone's mind is one question, "How do I land an internship?". Well, hopefully this section will help you start your application journey and be a guide for what you can do to help yourself best succeed. Here are a few key items with some simple tips: <br /> <br/>
+1. **Resumes**: The first thing you should do is work on your resume. If you don't have any relevant experience, that is okay — focus on building projects to put on your resume.  
+   - Use [**Jake's Resume Template**](https://www.overleaf.com/latex/templates/jakes-resume/syzfjbzwjncs) as a starting point. It’s a clean, ATS-friendly LaTeX format commonly used by students in CS.  
+<br /> 
+2. **Where to find Internships**:  
```

Comment:
https://github.com/vanshb03/Summer2026-Internships and https://github.com/vanshb03/New-Grad-2026 are better lists IMO